README.md

```diff
@@ -8,18 +8,25 @@ but there are currently no guarantees of stability or support.
 ## Usage
 ### Install the Proto Compiler
 This generator relies on the Protocol Buffer Compiler to [orchestrate] the
-client generation.
+client generation. Install protoc version 3.7 or later.
```

Comment:
Do we still need to have `protoc` installed now that we are using `grpc-tools`?

Reply:
I honestly have no idea. Where are we installing `grpc-tools`?

Reply:
It is a development dependency on gapic-generator. All the Rake tasks use it instead of protoc because it is much easier to use than installing protoc on the CI machines. So we removed the steps to install protoc for development, but they are still there for actually using the generator.

Reply:
Hmm. If the `grpc-tools` binaries are sufficient across platforms, it might be worth making it the preferred implementation for generator usage as well (i.e. make it a runtime dependency of `gapic-generator`). I think we can look into that separately, apart from this PR.